highfive/configs/rust-lang/rust.json

```diff
@@ -8,34 +8,56 @@
         "rustdoc": ["@QuietMisdreavus", "@steveklabnik", "@GuillaumeGomez", "@ollie27"]
     },
     "dirs": {
-        "src/doc":              ["doc"],
-        "src/liballoc":         ["libs"],
-        "src/libarena":         ["libs"],
-        "src/libbacktrace":     [],
-        "src/libcollections":   ["libs", "@Gankro"],
-        "src/libcore":          ["libs"],
-        "src/libcoretest":      ["libs"],
-        "src/libflate":         ["libs"],
-        "src/libfmt_macros":    ["libs"],
-        "src/libgetopts":       ["libs"],
-        "src/libgraphviz":      ["@pnkfelix"],
-        "src/liblibc":          ["libs"],
-        "src/liblog":           ["libs"],
-        "src/librand":          ["libs"],
-        "src/librbml":          ["libs", "@erickt"],
-        "src/librustc":         ["compiler"],
-        "src/librustdoc":       ["rustdoc"],
-        "src/libserialize":     ["libs", "@erickt"],
-        "src/libstd":           ["libs"],
-        "src/libsyntax":        ["syntax"],
-        "src/libterm":          ["libs"],
-        "src/libtest":          ["libs"],
-        "src/libunicode":       ["libs", "@SimonSapin"],
-        "src/rustbook":         ["doc"],
-        "src/ci":               ["infra-ci"],
-        ".azure-pipelines":     ["infra-ci"],
-        ".travis.yml":          ["infra-ci"],
-        "appveyor.yml":         ["infra-ci"]
+        "Cargo.lock":               ["@Mark-Simulacrum"],
+        "Cargo.toml":               ["@Mark-Simulacrum"],
+        "src/bootstrap":            ["@Mark-Simulacrum"],
+        "src/build_helper":         ["@Mark-Simulacrum"],
+        "src/doc":                  ["doc"],
+        "src/etc":                  ["@Mark-Simulacrum"],
+        "src/liballoc":             ["libs"],
+        "src/libarena":             ["libs"],
+        "src/libbacktrace":         [],
```

Comment:
The directory list looks seriously outdated.
Perhaps it needs to be regenerated.

Reply:
Yea, I had removed them, but decided this PR was pretty long and harder to review.  Submitted separately as #251.